### PR TITLE
SRE-112 | Remove unused memcache DELETEs

### PR DESCRIPTION
**Pull request by mszabo-wikia**
As part of a general audit of memcache DELETEs let's first remove clutter by getting rid of unused usages.
https://wikia-inc.atlassian.net/browse/SRE-112

### CODE REVIEW

File: includes/wikia/services/PageStatsService.class.php [hunk 1]
```diff
@@ -108,18 +108,10 @@ static function onArticleDeleteComplete(
...
-
 		wfProfileIn(__METHOD__);
 
 		wfDebug(__METHOD__ . ": page #{$this->pageId}\n");
 
-		// invalidate cached data from getCurrentRevision()
-		$wgMemc->delete($this->getKey('current-revision'));
-
-		// invalidate cached data from getPreviousEdits()
-		$wgMemc->delete($this->getKey('previous-edits'));
```
Comment (mszabo-wikia) [inline includes/wikia/services/PageStatsService.class.php:121]: The referred methods do not exist in code.

File: lib/Wikia/src/Tasks/Tasks/CreateNewWikiTask.php [hunk 2]
```diff
@@ -97,28 +97,18 @@ public function maintenance( $server ) {
...
-			\WikiFactory::removeVarById( $variable->cv_variable_id, $wgCityId );
-			\WikiFactory::clearCache( $wgCityId );
-		}
-
 		// SUS-3264 | set up events_local_users entries directly, instead of calling backend script
 		$this->info( "Setting up events_local_users table entries" );
 		\ListusersData::populateEventsLocalUsers( $wgCityId );
 
-		$wgMemc = wfGetMainCache();
-		$wgMemc->delete( \WikiFactory::getVarsKey( $wgCityId ) );
```
Comment (mszabo-wikia) [inline lib/Wikia/src/Tasks/Tasks/CreateNewWikiTask.php:112]: We do not set `wgReadOnly` variable while creating a wiki so this operation is redundant.

File: lib/Wikia/src/Tasks/Tasks/CreateNewWikiTask.php [hunk 3]
```diff
@@ -196,18 +186,6 @@ private function moveMainPage() {
...
-		 * clear skin cache for rt#63874
-		 *
-		 * @todo establish what $code is exactly in this case
-		 */
-		$parserMemc->delete( wfMemcKey( 'sidebar', $wgContLanguageCode ) );
-
-		$parserMemc->delete( wfMemcKey( 'quartzsidebar' ) );
-		$parserMemc->delete( wfMemcKey( 'navlinks' ) );
-		$parserMemc->delete( wfMemcKey( 'MonacoData' ) );
-		$parserMemc->delete( wfMemcKey( 'MonacoDataOld' ) );
```
Comment (mszabo-wikia) [inline lib/Wikia/src/Tasks/Tasks/CreateNewWikiTask.php:210]: These all relate to long-dead skins.

File: extensions/wikia/HAWelcome/HAWelcomeEdit.body.php [hunk 4]
```diff
@@ -1,161 +0,0 @@
-<?php
-
-class HAWelcomeEdit extends UnlistedSpecialPage {
```
Comment (mszabo-wikia) [inline extensions/wikia/HAWelcome/HAWelcomeEdit.body.php:3]: This class is not registered with autoloader nor MediaWiki special pages list.

File: extensions/wikia/HAWelcome/HAWelcome.i18n.php [hunk 5]
```diff
@@ -200,7 +198,6 @@
 
 M'ho peus ezhomm skoazell ha ma n'eus merour ebet amañ e c'hallit gweladenniñ ar [[wikia:Forum:Community Central Forum|foromoù Kreiz ar Gumuniezh]] $3",
 	'staffsig-text' => '[[{{ns:user}}:$1|$2]]<staff /> <small>([[w:forums|forom skoazellañ]] | [[w:sblog|blog]])</small>',
-	'hawelcomeedit' => 'HAWelcomeEdit',
```
Comment (mszabo-wikia) [inline extensions/wikia/HAWelcome/HAWelcome.i18n.php:203]: This message was only used by the removed `HAWelcomeEdit` page.

**Review by macbre — APPROVED** [review]

**Comment by mszabo-wikia** [comment]
CNW tests are green.

**Pull request merged**